drivers/common/opensensors.h

```diff
@@ -0,0 +1,30 @@
+#ifndef __OPENSENSORS_H__
+#define __OPENSENSORS_H__
+
+#define MAXSENSORS           10
```

Comment:
Update: sensors.h has been moved back to /bsp/boards, as general header file for all boards.
NUMOPENSENSORS is useless, so it has been removed.
Both opensensors and csensors create their own array with NUMSENSORS entries.
NUMSENSORS should be defined into board_info.h (platform dependant header file). This has been done for OpenMote-CC2538.
In case, NUMSENSORS is not defined on a platform-specific board_info.h, it is defined into sensors.h (ifndef clause) to a default value of 10.